lib/LANraragi/Utils/Database.pm

```diff
@@ -187,6 +190,14 @@ sub get_archive_json_multi (@ids) {
     return @archives;
 }
 
+sub get_archive_tags ($id) {
+    my %archive_info = get_archive($id);
+    return undef if ( !%archive_info );
+    return wantarray
```

Comment:
This is too much lmaoperl even for me here, I don't think subs should be able to return different types depending on what you ask for - especially without any kind of comments to specify the behavior 🤔  

Let's just return the normal tag string here, people can chain it to split_tags_to_array if needed.

Reply:
OK